### PR TITLE
Update working-with-the-npm-registry.md

### PR DESCRIPTION
**Pull request by kooya3**
### Why: 

Changing syntax for `npmrc` file using the code button. 

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@kooya3 Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/packages/working-with-a-github-packages-registry/working-with-the-npm-registry.md [hunk 1]
```diff
@@ -250,5 +250,5 @@ If your instance has subdomain isolation disabled:
 
 ## Using the official npm registry
 
-{% data variables.product.prodname_registry %} allows you to access the official npm registry at `registry.npmjs.com`, if your {% data variables.product.prodname_ghe_server %} administrator has enabled this feature. For more information, see [Connecting to the official npm registry](/admin/packages/configuring-package-ecosystem-support-for-your-enterprise#connecting-to-the-official-npm-registry).
+{% data variables.product.prodname_registry %} allows you to access the official npm registry at `registry.npmjs.com`, if your {% data variables.product.prodname_ghe_server %} administrator has enabled this feature. For more information, see [Connecting to the official `npm registry`](/admin/packages/configuring-package-ecosystem-support-for-your-enterprise#connecting-to-the-official-npm-registry).
```
Comment (subatoi) [inline content/packages/working-with-a-github-packages-registry/working-with-the-npm-registry.md:253]: We wouldn't accept this change as "npm registry" is how it's styled in the [docs that are being linked](https://docs.github.com/en/enterprise-server@3.11/admin/packages/configuring-package-ecosystem-support-for-your-enterprise#connecting-to-the-official-npm-registry), and in the [npm docs](https://docs.npmjs.com/).

File: content/packages/working-with-a-github-packages-registry/working-with-the-npm-registry.md [hunk 2]
```diff
@@ -225,7 +225,7 @@ By default, you can only use npm packages hosted on your enterprise, and you wil
 
 ### Installing packages from other organizations
 
-By default, you can only use {% data variables.product.prodname_registry %} packages from one organization. If you'd like to route package requests to multiple organizations and users, you can add additional lines to your _.npmrc_ file, replacing {% ifversion ghes or ghae %}`HOSTNAME` with the host name of {% data variables.location.product_location %} and {% endif %}`NAMESPACE` with the name of the personal account or organization {% ifversion packages-npm-v2 %}to which the package is scoped{% else %}that owns the repository containing the project{% endif %}.
+By default, you can only use {% data variables.product.prodname_registry %} packages from one organization. If you'd like to route package requests to multiple organizations and users, you can add additional lines to your `_.npmrc_ file`, replacing {% ifversion ghes or ghae %}`HOSTNAME` with the host name of {% data variables.location.product_location %} and {% endif %}`NAMESPACE` with the name of the personal account or organization {% ifversion packages-npm-v2 %}to which the package is scoped{% else %}that owns the repository containing the project{% endif %}.
```
Comment (subatoi) [inline content/packages/working-with-a-github-packages-registry/working-with-the-npm-registry.md:228]:
```suggestion
By default, you can only use {% data variables.product.prodname_registry %} packages from one organization. If you'd like to route package requests to multiple organizations and users, you can add additional lines to your `.npmrc` file, replacing {% ifversion ghes or ghae %}`HOSTNAME` with the host name of {% data variables.location.product_location %} and {% endif %}`NAMESPACE` with the name of the personal account or organization {% ifversion packages-npm-v2 %}to which the package is scoped{% else %}that owns the repository containing the project{% endif %}.
```

I agree this shouldn't be italicised, but we wouldn't use monospacing (backticks) around "file" as well.

**Review by subatoi — CHANGES_REQUESTED** [review]
Thanks for raising a PR! I'm happy to accept one part of this (with the suggested alternative change), but the other one I'm afraid we won't accept, so please revert that part.

**Comment by nguyenalex836** [comment]
@kooya3 Hello! 👋 We will be closing out this PR due to inactivity - but if you'd like to continue working on the suggestions we made above, please let us know 💛

**Pull request closed without merging**